autoload/neomake/makers/ft/yaml.vim

```diff
@@ -0,0 +1,12 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#yaml#EnabledMakers()
+    return ['yamllint']
+endfunction
+
+function! neomake#makers#ft#yaml#yamllint()
+    return {
+        \ 'args': ['-f', 'parsable'],
```

Comment:
@blampe The command here is `yamllint`: unlike`make`, it does not take a `-j` argument (this could be a good idea to propose upstream though).